presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add :func:`reduce_agg` aggregate function.
+* Add :func:`milliseconds` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
```

Comment:
I don't think it's super clear what "predicate pushdown" means.  I think it would be clearer to say `Push filters on window function partition symbols below window operators.`  (I assume that's what this is about)

Reply:
We shouldn't mention "partition symbols" at all since that is an internal detail of the planner. This should be explained in end-user terms.